test/integration/framework/framework.go

```diff
@@ -347,3 +347,23 @@ func ExpectResourceFlavorToBeDeleted(ctx context.Context, k8sClient client.Clien
 		return k8sClient.Get(ctx, client.ObjectKeyFromObject(rf), &newRF)
 	}, Timeout, Interval).Should(testing.BeNotFoundError())
 }
+
+func ExpectLocalQueueToBeDeleted(ctx context.Context, k8sClient client.Client, lq *kueue.LocalQueue, deleteLq bool) {
+	if deleteLq {
+		gomega.Expect(DeleteLocalQueue(ctx, k8sClient, lq)).To(gomega.Succeed())
+	}
+	gomega.Eventually(func() error {
```

Comment:
Are you having issues if you don't do wait?

We have this wait in objects that can be blocked during deletion, like ClusterQueue.

Reply:
> Are you having issues if you don't do wait?

I have no problems worth mentioning.

> We have this wait in objects that can be blocked during deletion, like ClusterQueue.

I see. I would remove the `ExpectLocalQueueToBeDeleted` function and the `ExpectWorkloadToBeDeleted ` function.

Reply:
Done.